handbook/company/customer-first.md

```diff
@@ -13,7 +13,7 @@ This is the area of responsibility for each organization:
 
 * Sales is responsible for establishing, retaining, and growing our commercial relationship with customers.
 * Customer Engineering (aka CE) is responsible for educating our users on how to use Sourcegraph, providing proactive technical expertise, and is overall accountable for our customers’ success, usage, and adoption of our products.
-* Effective 2021-08-08, Customer Support (aka CS) is responsible for satisfying support SLAs for customers pre-sales (1 hour) and post-sales (defined contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas)) between 2am-5pm Pacific Time Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22). Between now and 2021-08-08, as the CS team hires and onboards additional folks, it will adhere to a 24 hour SLA for all pre-sales issues between 9am-5pm PT Monday-Friday.
+* Customer Support (aka CS) is responsible for satisfying support SLAs for customers pre-sales (1 hour) and post-sales (defined contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas)) between 2am-5pm PT (9:00-0:00 UTC) Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22).
```

Comment:
How did the 2am - 5pm PT (9:00-0:00 UTC) time commitment come about? Also if I was a customer, I think the 2am-5pm PT being first and the (9:00-0:00 UTC)  in parentheses shows a bias towards the west coast as opposed to being more general. 

I suggest switching the placement of both or even removing the PT. Unless there is a certain reason for it that I am not privy to.

Reply:
Great question! The reason is that our SLA terms in our contracts are stated in pacific. And this expands upon our contractual SLA.

Reply:
But I think it's fair to just say UTC and not overthink it (on my part)

Reply:
Thanks for the insight about the SLA terms being stated in Pacific time!